src/main/java/com/redhat/quarkus/mandrel/collector/report/endpoints/ImageStatsResource.java

```diff
@@ -85,8 +85,21 @@ public class ImageStatsResource {
     // TODO: Paging
     @RolesAllowed("token_read")
     @GET
-    public ImageStats[] list() {
-        return collection.getAll();
+    public ImageStats[] list(
+            @QueryParam("tag") String tag,
+            @QueryParam("imgName") String imgName,
+            @QueryParam("olderThan") String olderThan,
+            @QueryParam("newerThan") String newerThan) {
```

Comment:
Those time values should be documented in terms of time zone.

Reply:
Added a javadoc block on the method.